activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -385,6 +385,9 @@ def load_schema(db_config, format = ActiveRecord.schema_format, file = nil) # :n
           load(file)
         when :sql
           structure_load(db_config, file)
+        when :sql_filesystem_versions
+          migration_operation = ActiveRecord::Migration::SqlFilesystemVersions.new(db_config, migration_connection_pool, self)
```

Comment:
as per comment above - I wanted to somehow break out of this namespace to unit test this thing without baggage of `DatabaseTasks` module; any better ideas? 🤔